parserv3.go

```diff
@@ -351,6 +351,13 @@ func parseSecAttributesV3(context string, lines []string, index *int) (string, *
 		search = []string{authorizationURL, in}
 	case secAccessCodeAttr:
 		search = []string{tokenURL, authorizationURL, in}
+	case secBearerAuthAttr:
+		scheme := spec.SecurityScheme{
+			Type:         "http",
+			Scheme:       "bearer",
+			BearerFormat: "JWT",
```

Comment:
We allow only jwt bearers ?

Reply:
According to the OpenAPI 3.0 specification, Bearer Token must be used as defined.

If the user wants to use another security scheme, they need to append it under the security section in the YAML configuration.
 ![image](https://github.com/user-attachments/assets/f4a739fe-12a2-4c8b-b197-9674c76eca8a)